packages/noports_core/test/sshnp/util/sshrvd_channel/sshrvd_channel_test.dart

```diff
@@ -134,12 +152,27 @@ void main() {
       when(() => mockParams.host).thenReturn('234.234.234.234');
       when(() => mockParams.port).thenReturn(135);
 
-      await expectLater(stubbedSshrvdChannel.callInitialization(), completes);
+      await expectLater(stubbedSshrvdChannel.initialize(), completes);
 
       expect(stubbedSshrvdChannel.host, '234.234.234.234');
       expect(stubbedSshrvdChannel.port, 135);
     }); // test Initialization - non-sshrvd host
 
+    test('Initialization completes - sshrvd host', () async {
+      /// Set the required parameters
+      whenInitializationWithSshrvdHost();
```

Comment:
Added an init completion test, which checks if `completeInitialization()` is called when `callInitialization()` is called ... this only needs to be tested in certain places, usually impl classes, but initialization only occurs in the base class for sshrvd channel, so we can call and test for it in the base class.